spec/models/manageiq/providers/ansible_tower/automation_manager/credential_mixin_spec.rb

```diff
@@ -0,0 +1,19 @@
+class DummyCredential < ::Authentication
```

Comment:
Please don't create class constants in the test environment.  They are not limited in scope to this test and have conflicted in the past and troubleshooting is painful.  Instead, if needed:
```ruby
let(:test_class) do
  class.new(::Authentication) do
    ATTRIBUTES = []
    etc...
  end
end
```